src/components/Operation.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.